frame/cosmwasm/src/instrument.rs

```diff
@@ -0,0 +1,26 @@
+use codec::{Decode, Encode};
+use parity_wasm::elements::Module;
+use scale_info::TypeInfo;
+use wasm_instrument::gas_metering::{self, Rules};
+
+/// Errors likely to happen while instrumenting a code.
+#[derive(Clone, Debug, Encode, Decode, TypeInfo)]
+pub enum InstrumentationError {
+	GasMeteringInjection,
+	StackHeightLimitingInjection,
```

Comment:
These names are a bit technical, would like to see comments describing what is wrong with the module if these are encountered, or fields with more info

Reply:
We now log the error at the node level.